developer/api-plugin-helper-http_server.md

```diff
@@ -0,0 +1,66 @@
+# Http Server Plugin Helper API
+
+`server_http` helper creates http server.
+
+Here is the code example with `server_http` helper:
+
+```rb
+require 'fluent/plugin/input'
+
+module Fluent::Plugin
+  class ExampleInput < Input
+    Fluent::Plugin.register_output('example', self)
+
+    # 1. load http_server helper
+    helpers :http_server
+
+    config_param :host, :string
+    config_param :port, :integer
+
+    def start
+      super
+
+      # 2. create and start http server
+      create_http_server(addr: @bind, port: @port, logger: log) do |serv|
+        # define endpoint `/hello` with GET method
+        serv.get('/hello') { [200, { 'Content-Type' => 'text/plain' }, 'hello!'] }
+      end
+    end
+  end
+end
+```
+
+Launched http server is managed by the helper. No need to stop http server
+in plugin's `stop` method. The plugin stops launched http server automatically.
+
+## Methods
+
+### create\_http\_server(addr:, port:, logger:, default_app: nil, &block)
+
+This method creats and runs http server with given routes which are defined in `&block`.
+
+- `addr`: Adderess to listen to
+- `port`: Port to listen to
+- `logger`: Logger which is used in server helper
+- `default_app`: Use this object when server received a request whose path is not registered. This object must have `#call` or be a `Proc` object.
+
+** Define other HTTP methods
```

Comment:
should mention return value and typical methods of `req`.

Reply:
Added 
https://github.com/fluent/fluentd-docs-gitbook/pull/58/commits/593b04a304d6bab329dd58e3a80323a58de5b7e3